docs/agent-api.asciidoc

```diff
@@ -86,7 +86,8 @@ deprecated[4.1.x,Tags have been replaced with labels], Please use <<apm-add-labe
 apm.addLabels({ [name]: value })
 ----
 
-Set labels on transactions and errors.
+Set labels on transactions and errors. 
+Starting with APM Server 7.6+, the labels are added to spans as well.
```

Comment:
this happen on the server right? For global labels?

Reply:
That's correct!

Reply:
Do you want to link to the server docs? just an idea

Reply:
I don't think that's necessary since the user shouldn't need to care about how it works behind the scene.